### PR TITLE
Cls2 382 archived objectives

### PR DESCRIPTION
**Pull request by CaitBarnard**
## Description of change

Page for showing archived objectives.

## Test instructions

URL: /companies/{companyId}/account-management/objective/archived

## Screenshots
Screenshot 2023-08-17 at 14 24 30
Page shouldn't be able to be accessed if there are no archived objectives but just in case:
Screenshot 2023-08-17 at 14 24 42

_Add a screenshot_

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import { useParams } from 'react-router-dom'
+
+import { GridCol, GridRow } from 'govuk-react'
+import { H4 } from 'govuk-react'
+import { FONT_SIZE, SPACING } from '@govuk-react/constants'
+import styled from 'styled-components'
+import { DefaultLayout, Metadata } from '../../../components'
+import { buildCompanyBreadcrumbs } from '../utils'
+import { DARK_GREY, GREY_2 } from '../../../utils/colours'
+import urls from '../../../../lib/urls'
```
Comment (paulgain) [inline src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx:11]: It's best to organise these imports into two sections with a line break between the two:
1. external dependencies 
3. internal dependencies.

**Review by paulgain — APPROVED** [review]

File: src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx (hunk 1 above)
Comment (CaitBarnard) [reply src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx]: Thanks! I've reordered, hopefully correctly this time :)

Comment (paulgain) [reply src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx]: @CaitBarnard nearly :-)

The following line is in section 2, it should be in section 1.

```js
import styled from 'styled-components'
````

File: src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import { GridCol, GridRow, H4 } from 'govuk-react'
+import { FONT_SIZE, SPACING } from '@govuk-react/constants'
+
+import styled from 'styled-components'
```
Comment (cgsunkel) [inline src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx:6]: This should be with the other third party imports

File: src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    {
+      label: 'Progress',
+      value: `${objective.progress}%`,
+    },
+    {
+      label: 'Adviser',
+      value: objective.modifiedBy?.name,
+    },
+  ]
+  return rows
```
Comment (cgsunkel) [inline src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx:56]: It should be possible to use an implicit return here

**Review by cgsunkel — APPROVED** [review]
I have a couple of very minor comments but looks good otherwise

File: src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx (hunk 3 above)
Comment (CaitBarnard) [reply src/client/modules/Companies/AccountManagement/ArchivedObjectives.jsx]: It wasn't quite happy without a return but I've removed the variable part as it was unnecessary. Thanks!

**Pull request merged**